### ISSUE TITLE
Please improve installation notes

### ISSUE THREAD

**Issue by re-mx** [flycheck/flycheck-rust#63]
Hello:

In you README:
```
(add-hook 'flycheck-mode-hook #'flycheck-rust-setup)
```
In some cases it seems unreliable.

My advice:
```
(with-eval-after-load 'rust-mode
    (add-hook 'flycheck-mode-hook #'flycheck-rust-setup))
```

Sorry for my poor English.

Thanks.

**Comment by fmdkdd** [comment]
I think I wrote it that way since users may use `use-package` or something else.  But it may be best to provide more robust copy-and-paste instructions.

Fixed by 089330ac3bb49525ae6a8607a9ea978560ccf721

**Comment by re-mx** [comment]
Thank you.
